build/commands/lib/config.js

```diff
@@ -328,6 +329,10 @@ const Config = function () {
   ]
 }
 
+Config.prototype.useNoGnGen = function () {
+  return this.no_gn_gen != null
```

Comment:
`!= null` is unusual. this should either be `return this.no_gn_gen` or `return this.no_gn_gen === true`

`.env` values are parsed via `JSON.parse`, so true/false will become a JS boolean here.

Reply:
or `return !!this.no_gn_gen` which will automatically handle:
 0, >0, undefined, null, false, true
 
 wont handle 'false' or 'true' though

Reply:
`== null` or `!= null` is loose equality so handles null and undefined